server/functions/countShips.js

```diff
@@ -7,8 +7,50 @@ function countShips(cellId, arrayIn) {
     return !(i % 10) ? prev.concat([a.slice(i, i + 10)]) : prev
   }, []);
 
-  // console.log('array', arrayOut[0][9])
-
+  //-------------------------------------------------
+  let count = 0;
+//vertical
+  for (let i = 0; i < 10; i++) {
+    if (i !== +coordX) {
```

Comment:
Instead of if inside if, you can write
```
if ((i !== +coordX) && ((arrayOut[i][coordY].content === 'ship') || (arrayOut[i][coordY].content === 'wreck')))
```

Reply:
ok